api/organisations/tests/test_models.py

```diff
@@ -58,3 +60,80 @@ def test_max_seats_set_as_one_if_subscription_has_no_subscription_id(self):
 
         # Then
         assert subscription.max_seats == 1
+
+
+@pytest.mark.django_db
+def test_creating_a_subscription_calls_mailer_lite_subscribe_organisation(mocker):
+    # Given
+    organisation = Organisation.objects.create(name="Test org")
+    mocked_mailer_lite = mocker.patch("organisations.models.mailer_lite")
+
+    # When
+    Subscription.objects.create(organisation=organisation)
+
+    # Then
+    mocked_mailer_lite.subscribe_organisation.assert_called_with(organisation.id)
+
+
+@pytest.mark.django_db
```

Comment:
There is a `db` fixture available in the `pytest-django` module that achieves the same thing as this decorator.

Reply:
There are still a few instances of this decorator but it's not important - more a suggestion than a request. I'll resolve this comment.